packages/strapi-plugin-users-permissions/config/policies/rateLimit.js

```diff
@@ -0,0 +1,12 @@
+const RateLimit = require('koa2-ratelimit').RateLimit;
+
+module.exports = async (ctx, next) => {
+  const message = ctx.request.admin ? [{ messages: [{ id: 'Auth.form.error.ratelimit' }] }] : 'Too many attempts, please try again in a minute.';
+
+  return RateLimit.middleware({
```

Comment:
It could be awesome if we could be able to edit the settings from the configurations file `./environment/*/request.json`. What do you think?

Reply:
@Aurelsicoko yes please